src/tools/kdb/gen/templates/collect.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-# Copyright (C) 2019 Elektra Initiative.
+# Copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```

Comment:
@a-kraschitzer 
All the files in `src/tools/kdb/gen/templates` are templates for client side code generation. This means the copyright line must make sense outside of the Elektra repo as well. 

The copyright should be changed in all these files (and the corresponding test files) to something like:

```sh
Copyright (c) 2019-2021 Elektra Initiative (https://www.libelektra.org)
```

The year could be omitted, but I think the generated code should reflect from which version of Elektra it originates. More importantly the BSD Zero Clause License used for the template files is different from the BSD 3-Clause License used for the rest of the project (i.e. the one in LICENSE.md).

---

PS. this file should also contain the full license and the title "BSD Zero Clause License" like the `*.mustache` files.

Reply:
Can you please create a new issue or even better a PR fixing the problem? I think nobody saw it here.

> The year could be omitted

Yes, please omit the year. One place to update every year is enough.

Reply:
> One place to update every year is enough.

We could add a basic GitHub Action that only checks whether a set of predefined locations (e.g. line 3 of `LICENSE.md`) contains the current year. For example via:
```sh
sed '3!d' LICENSE.md | grep -Fq $(date +%Y)
```
It would fail for the first PR every year, so we can't forget to update the year.

Reply:
> We could add a basic

There are dozens of problems that happen frequently and have severe impact but could be easily detected by CI. So we should not waste time doing things in CI that happen rarely and have basically no impact and could be avoided in the first place by simply not introducing the problem (not adding the year).

> GitHub Action

Exactly the same could be done in Jenkins, which is fully under our control.

Reply:
> simply not introducing the problem (not adding the year).

Should've been more clear. I'm fine with leaving the year out of the template files. But it has to be there in LICENSE.md and it is quite easy to forget about it.

> Exactly the same could be done in Jenkins,

Yes, but IMO creating a GitHub Action is easier than a Jenkins job.

> which is fully under our control.

True, but I don't think this job so important that we need to be fully in control of it ;)

Reply:
This is what you mean :wink:

For this particular year we already have a reminder in doc/todo/RELEASE.md

We could add the check what you suggested in the release pipeline but imho it is not worth failing a release for a wrong date.